### ISSUE TITLE
Server shutdowns randomly

### ISSUE THREAD

**Issue by ses1112** [typescript-language-server/typescript-language-server#471]
Hello,

I have a strange problem. 

The situation:
We use Netbeans and connected this lsp server with the Netbeans lsp-client. 

The LSP-Server is started with following command:
C:\path\to\node\bundled_nodejs\node.exe node_modules/typescript-language-server/lib/cli.js --stdio --log-level 4

Now it sometimes occurs that the process of this lsp server is excited. Therefor Netbeans starts the lsp again, but that takes a little while. 

Has somebody any idea why the server shutdowns? Is there any flag where it can be analyzed?

**Comment by rchl** [comment]
I don't know how to do it in your editor but if you pass those options to the server then `.log` directory will be created in the current project:

```
    "initializationOptions": {
        "logVerbosity": "verbose",`
    }
```

It would also be worth looking at LSP communication log which I'm also not sure how to enable in your editor but there should be some way.

**Comment by ses1112** [comment]
Thanks for your advices. They helped a lot.

Netbeans shutdowns the lsp server if it wasn't used the last 10 minutes. Unfortunately, I only found this code now. This is now fixed.

The next problem was that the first request to the lsp server tooks sometimes more than 50 seconds. In the log the lsp server added some file watchers and we have a big project so this took some time. I implemented that instantly after the server is started a pseudo request is sent which opens randomly a file. The server can load everything and the first "real" request can be responded fast.
